src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -340,12 +340,13 @@ export class ExposureNotificationService {
         return finalize();
       }
       return finalize({needsSubmission: await this.calculateNeedsSubmission()});
-    } else if (
-      currentStatus.type === 'exposed' &&
-      currentStatus.summary.daysSinceLastExposure >= EXPOSURE_NOTIFICATION_CYCLE
-    ) {
-      this.exposureStatus.set({type: 'monitoring', lastChecked: currentStatus.lastChecked});
-      return finalize();
+    } else if (currentStatus.type === 'exposed') {
+      const today = getCurrentDate();
+      const lastExposureAt = new Date(currentStatus.summary.lastExposureTimestamp || today.getTime());
```

Comment:
The `||` here is just a safety check, just in case people upgrades from previous build. So that, it won't crash. If people is in exposed state in previous build, they will be in exposed state forever even after upgrading to this build until they are being exposed again.